regulations/static/regulations/js/source/views/main/main-view.js

```diff
@@ -249,6 +251,24 @@ var MainView = Backbone.View.extend({
 
     setHandlers: function() {
         this.applyTablePlugin();
+        this.applyClipboardPlugin();
+    },
+
+    applyClipboardPlugin: function() {
+        // Create anchor tag for copy to clipboard
+        this.$el.find('.poster').before(
```

Comment:
I think this selector should be more generic so that this functionality can be extended to other things.  Maybe add a class "copyable" or something like data-clipboard-enabled="true".